root.tf

```diff
@@ -484,6 +493,36 @@ module "export_step_function" {
   })
 }
 
+module "new_export_bucket" {
+  source            = "./da-terraform-modules/s3"
+  bucket_name       = "tdr-export-${local.environment}"
+  kms_key_arn       = module.s3_external_kms_key.kms_key_arn
+  common_tags       = local.common_tags
+  use_random_suffix = true
+}
+
+module "new_export_bucket_judgment" {
+  source            = "./da-terraform-modules/s3"
+  bucket_name       = "tdr-export-judgment-${local.environment}"
+  kms_key_arn       = module.s3_external_kms_key.kms_key_arn
+  common_tags       = local.common_tags
+  use_random_suffix = true
+}
+
+module "export_sns_notifications_topic" {
```

Comment:
This topic should be made more generic so not just export notification can be sent to it.

Suggest changing:

- module name to: `external_sns_notifications_topic`
- `topic_name` local to: `local.external_notifications_topic_name`
- `sns_policy` template file name to: `.../sns/external_notifications_policy ...`

Reply:
Fair points, I've changed them all.